generate_testdata.sh

```diff
@@ -61,9 +61,15 @@ scaffold_test_project() {
             header_text 'Creating APIs ...'
             $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
             $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
-            $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
-            $kb create webhook --group crew --version v1 --kind FirstMate --conversion
             $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
+            $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
+            if [ $plugin_flag == "go/v3-alpha" ]; then
+                header_text 'Creating API for conversion webhook ...'
+                $kb create api --group crew --version v2 --kind FirstMate --controller=false --resource=true --make=false
+                $kb create webhook --group crew --version v2 --kind FirstMate --hub=v1 --spoke=v2
```

Comment:
I did not find the scaffold in the project-v3 with the --hub/spoke content. Should not it change the scaffolds?

Reply:
Have modified the script. The scaffold appears now.